kafka-ui-api/src/main/java/com/provectus/kafka/ui/util/KafkaClientSslPropertiesUtil.java

```diff
@@ -0,0 +1,34 @@
+package com.provectus.kafka.ui.util;
+
+import com.provectus.kafka.ui.config.ClustersProperties;
+import java.util.Properties;
+import javax.annotation.Nullable;
+import org.apache.kafka.common.config.SslConfigs;
+
+public final class KafkaClientSslPropertiesUtil {
+
+  private KafkaClientSslPropertiesUtil() {
+  }
+
+  public static void addKafkaSslProperties(@Nullable ClustersProperties.TruststoreConfig truststoreConfig,
+                                           Properties sink) {
+    if (truststoreConfig == null) {
+      return;
+    }
+
+    if (truststoreConfig.getTruststoreLocation() == null) {
+      return;
+    }
```

Comment:
what if `verifySsl` set to false ? Looks like  truststoreConfig.isVerifySsl() check should go before this..